agent/bench-scripts/postprocess/trafficgen-postprocess

```diff
@@ -29,81 +28,78 @@ my $script_name = basename($0);
+my %efficiency;	# a hash of throughput-per-resource-type:array-of-throughput-per-resource-values
+				# for example $efficincy{Mframes_cpu[0..1]}
 my @Mframes_cpu;# each array element contains a hash with:
-		# hostname:
-	    	# port_id:
-	    	# role:
-	    	# description:
-	    	# timeseries: a hash of timestamp,value elements
-
-my %latency;    # a hash of latency-type:array-of-latency-values,
-		# for example $latency{usec[0..1]}
-		#
-		# each array element contains a hash with:
-	    	# hostname:
-	    	# port_id:
-	    	# role:
-	    	# description:
-my @usec_avg;   # 
+				# hostname:
+				# port_id:
+				# role:
+				# description:
+				# timeseries: a hash of timestamp,value elements
+
+my %latency;	# a hash of latency-type:array-of-latency-values,
+				# for example $latency{usec[0..1]}
+				#
+				# each array element contains a hash with:
+				# hostname:
+				# port_id:
+				# role:
+				# description:
+my @usec_avg;
 my @usec_min;
 my @usec_max;
 
 
-my %throughput; # a hash of throughput-type:array-of-throughput-values,
-		# for example $throughput{Mpps[0..1]
+my %throughput;	# a hash of throughput-type:array-of-throughput-values,
+				# for example $throughput{Mpps[0..1]
 
-my @rx_Mpps;    #
-my @tx_Mpps;    # each array element contains a hash with:
-	    	# hostname:
-	    	# port_id:
-	    	# role: packet-generator
-	    	# value: the number value for millions of pckets (frames) per second
+my @rx_Mpps;	#
+my @tx_Mpps;	# each array element contains a hash with:
+				# hostname:
+				# port_id:
+				# role: packet-generator
+				# value: the number value for millions of pckets (frames) per second
 
 my @rx_L1_Gbps; #
 my @tx_L1_Gbps; # each array element contains a hash with:
-	    	# hostname:
-	    	# port_id:
-	    	# role: packet-generator
-	    	# the number value for 10^9 bits per second for physical Ethernet (layer 1, including CRC and padding)
-	    	
+				# hostname:
+				# port_id:
+				# role: packet-generator
+				# the number value for 10^9 bits per second for physical Ethernet (layer 1, including CRC and padding)
+
 my @rx_L2_Gbps; #
 my @tx_L2_Gbps; # each array element contains a hash with:
-	    	# hostname:
-	    	# port_id:
-	    	# role: packet-generator
-	    	# the number value for 10^9 bits per second for physical Ethernet (layer 1, including CRC and padding)
-
-sub mean {
-    return sum(@_)/@_;
-}
+				# hostname:
+				# port_id:
+				# role: packet-generator
+				# the number value for 10^9 bits per second for physical Ethernet (layer 1, including CRC and padding)
 
 my $script = basename($0);
 my $dir = $ARGV[0];
-my $iteration = $ARGV[1];
-my $max_loss_pct = $ARGV[2];
-my $benchmark_version = $ARGV[3];
-my $tool_label_pattern = $ARGV[4];
-my $tool_group = $ARGV[5];
+my $controller_host = $ARGV[1];
+my $rate_arg = $ARGV[2];
+my $frame_size_arg = $ARGV[3];
+my $num_flows_arg = $ARGV[4];
+my $traffic_profile_arg = $ARGV[5];
+my $benchmark_version = $ARGV[6];
```

Comment:
I don't know enough about trafficgen to answer what it will generate.  By "pbench" I think you mean the `pbench-trafficgen` script, which offers a `--one-shot` parameter (see [line 237](https://github.com/portante/pbench/blob/go-back-fix-trafficgen/agent/bench-scripts/pbench-trafficgen#L237)).  So then the question becomes are there cases where people use `pbench-trafficgen` with `--one-shot`, and I don't know the answer to that question immediately.

Reply:
Apparently, [there _are_](https://docs.google.com/document/d/1FokNXV8ZBAzac78fUoqdUBWXTxDWIEBP2YjWNpRioT8/edit#heading=h.6c70xeopatiu) such cases.